tutorials/pl.json

```diff
@@ -0,0 +1,4 @@
+{
+    "home": "<h2>Samouczek: Widok Fabryki/h2>\n<p>Jesteś obecnie w widoku fabryki. Tutaj możesz zobaczyć i zarządzać jej układem.</p>\n<ul>\n\t<li><b>Prawa część</b> pokazuje globalny magazyn fabryki. Możesz <b>wyróżnić</b> niektóre zasoby jako ulubione używając ikony <StarIcon /> icon.</li>\n\t<li><b>Dolna część</b> zawiera listę wyposażenia, które możesz zbudować. Kliknij na jedno z nich, aby uruchomić tryb budowy.</li>\n\t<li><b>Główna część</b> pokazuje plan twojej fabryki. Możesz kliknąć <b>prawym przcyskiem myszy</b> na elemencie wyposażenia, aby wyświetlić szczegóły. Następnie możesz użyć ikony <MoveIcon /> (or <kbd>M</kbd>) aby przenieść sprzęt, albo ikony <DeleteIcon /> (or <kbd>D</kbd>) aby go usunąć/sprzedać.</li>\n</ul>",
```

Comment:
The `(or <kbd>M</kbd>)` means "or the keyboard key M" (it will be shown as a keyboard key visually), I believe we should translate the word "or" here. Same for `(or <kbd>D</kbd>)`